### DISCUSSION TITLE
An input model is working on Andes 1.5.11 but not on 1.7.3

### DISCUSSION THREAD

**Discussion by TheASingh** [CURENT/andes#349] [Q&A]
Hi All,

An ieee14 bus input model is working fine on Andes version 1.5.11. But when I upgraded Andes to 1.7.3, I'm getting below error:

**Python has stopped working.
A problem caused the program to stop working correctly. Windows will close the program and notify you if a solution is available.**

Can you please guide me why an input model is running on Andes 1.5.11 but not on 1.7.3 ?

**Comment by cuihantao** [comment]
Can you upload the case file? That will help to troubleshoot.

**Comment by TheASingh** [comment]
We currently put this requirement on hold and now testing it with 1.5.11.

**Reply by cuihantao** [reply]
Sure. I understand if sharing is not an option for now. Let me know at any time if you have a minimum case to reproduce the issue.

**Reply by cuihantao** [reply]
My email address is cuihantao@gmail.com if you'd like me to take a look. Thanks!

**Reply by TheASingh** [reply]
I tried my input model on latest andes versions (1.7.6 and 1.7.7).  I'm able to run.

Thanks

**Comment by cuihantao** [comment]
Thanks. Interesting. Did you get a chance to hunt down which model was causing the issue?

**Reply by TheASingh** [reply]
I tried to compare my input model with the original ieee14 bus system's input model. There was no difference in any of the columns of all models. May be its due to the data.

But not sure.
